.travis.yml

```diff
@@ -65,6 +72,15 @@ matrix:
           services: [docker]
           env: BUILD=tests,wheels,release
 
+        - os: linux
+          dist: xenial
+          branches: {only: [releases]}
+          sudo: required
+          language: python
+          python: "3.7"
+          services: [docker]
+          env: BUILD=tests,wheels,release
```

Comment:
Drop "wheels,release" from `BUILD`.

Reply:
Could also be the same for other python versions, no ?

Reply:
It's the same for other versions. There's only one target with `BUILD=tests,wheels,release` (3.5) which builds the wheels and releases them.  All other targets (except macos) only run tests.  So 3.7 target should also just run tests.